### ISSUE TITLE
Tables aren't created if activated via wp-cli

### ISSUE THREAD

**Issue by pjeby** [ericmann/wp-session-manager#65]
If you activate the plugin via `wp plugin activate`, the DB table isn't created, and subsequent attempts to use the plugin generate errors.  Adding this line to the main plugin file fixed the issue for me:

```php
register_activation_hook(__FILE__, ['EAMann\WPSession\DatabaseHandler', 'create_table']);
```

**Comment by ericmann** [comment]
Would you mind to add that as a PR? I can merge it quickly ;-)

**Comment by ericmann** [comment]
Fixed in #66.
